src/index.ts

```diff
@@ -1535,8 +1535,10 @@ export class CachedMerklePatriciaTree<K, V> extends MerklePatriciaTree<K, V> {
   // depth 1.
   private maxCacheDepth: number;
 
-  constructor(depth = 6) {
-    super();
+  constructor(
+      options: MerklePatriciaTreeOptions<K, V> = {putCanDelete: true},
+      depth = 6) {
```

Comment:
You should use a public auto-property here:
```
constructor(options, public maxCacheDepth = 6);
```
Is there some reason maxCacheDepth is private anyway?